### ISSUE TITLE
Wondering if there will be a 1.18.2 version?

### ISSUE THREAD

**Issue by IdrisQe** [vladmarica/stopwatch#1]
Just wanting to ask if a 1.18.2 version is planned to happen. I've been trying to find a profiler for loading times but everything is stuck between 1.12 and 1.16, this seems to be both the most recent and most up-to-date one, but the problem is my current modpack which I want a profiler to find the source of startup freezes and long loading times, is in 1.18.2.

I'm not trying to beg for it to be updated ASAP or anything, I just want to know since it's legitimately a super useful thing and should be a feature of Forge itself honestly.

**Comment by vladmarica** [comment]
Yes, I am planning on updating this mod to 1.18.2 once the 1.16.5 version is complete. I haven't had time to work on this mod recently since I've working on other mods, but I hope to get back to it soon.
